app/routes/_landing.network/route.tsx

```diff
@@ -91,6 +92,9 @@ export const loader: LoaderFunction = async ({ request }) => {
   const dailyNetworkRelays = await getRelays("network", today, today)
   const weeklyNetworkRelays = await getRelays("network", week, today)
   const monthlyNetworkRelays = await getRelays("network", month, today)
+  const dailyNetworkRelaysPerMonth = await getRelaysPerPeriod("network", 30)
```

Comment:
I think it's unnecessary to fetch all of the data when it's not needed unless asked for by the user. I think we should fetch the initial default data, and then move the rest to an action? and only fetch them once the user asks for the data.